src/libraries/Common/tests/System/Net/Prerequisites/NetCoreServer/NetCoreServer.csproj

```diff
@@ -5,6 +5,7 @@
     <AspNetCoreHostingModel>InProcess</AspNetCoreHostingModel>
     <OutputType>Exe</OutputType>
     <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
+    <StaticWebAssetsEnabled>false</StaticWebAssetsEnabled>
```

Comment:
What is this?

Reply:
https://github.com/dotnet/runtime/pull/57143#issuecomment-896214913

Reply:
cc @javiercn

Reply:
If it's fixed in the SDK do we need to remove these now?

Reply:
yes, I'll revert the various workarounds